Cargo.toml

```diff
@@ -111,7 +111,9 @@ surfman = { version = "0.9", features = ["chains", "sm-angle", "sm-angle-default
 syn = { version = "2", default-features = false, features = ["clone-impls", "derive", "parsing"] }
 synstructure = "0.13"
 thin-vec = "0.2.13"
+# TODO: Migrate Servo from time 0.1 to 0.3 so that we don't need to build both versions.
 time = "0.1.41"
+time_03 = { package = "time", version = "0.3 ", features = ["std", "local-offset"] }
```

Comment:
Issue #30150 suggests replacing the old "time" dependency with one of these alternatives:

* std::time - I don't think this makes it possible to format a filesystem timestamp into a readable datetime;
* chrono - this does implement `From<std::time::SystemTime>` so it might be suitable, but I don't see "chrono" already listed in the main Cargo.toml file;
* time (latest version) - this is what I've used already, by adding the alias "time_03" to the Cargo.toml file.

Can you (or @mrobinson) tell me which is the preferred approach?

Reply:
Martin Robinson has confirmed (in Zulip thread "Directory Listing Feature") that using the "time" v0.3 crate is acceptable, so the code which handles file/directory last-modified formatting shouldn't need any further changes.